R/gCrisprTools.R

```diff
@@ -10,7 +10,7 @@ NULL
 ##' @import limma
 ##' @import RobustRankAggreg
 ##' @import ggplot2
-##' @import PANTHER.db
+##' @import sparrow
```

Comment:
Since you're only using a small subset of the sparrow package, you may want to consider not importing the whole package. Just keep the package listed in the Imports of your DESCRIPTION file and prefix the functions you do use with `sparrow::functionName(...)` ... like you are (mostly) doing already.